dask_jobqueue/tests/test_remote_slurm.py

```diff
@@ -0,0 +1,56 @@
+import sys
+
+import pytest
+from distributed import Client
+
+from dask_jobqueue.remote_slurm import RemoteSLURMCluster
+
+TEST_API_URL = "http://a-url.com/slurm"
+
+
+@pytest.mark.env("slurm")
+@pytest.mark.asyncio
+async def test_end_to_end(aresponses, loop):
+    aresponses.add(TEST_API_URL, "/jobs/submit", "POST", {"job_id": 1001})
+    aresponses.add(
+        TEST_API_URL,
+        "/job/1001",
+        "DELETE",
+        aresponses.Response(status=200),
+    )
+
+    async with RemoteSLURMCluster(
+        api_url=TEST_API_URL,
+        walltime="00:02:00",
+        processes=4,
+        cores=8,
+        memory="28GB",
+        n_workers=1,
+        job_extra=["--gpus 2"],
+        loop=loop,
```

Comment:
You probably need the `asynchronous=True` kwarg here also.

Reply:
Thanks, missed that one.